services/workflows-service/scripts/alerts/generate-alerts.ts

```diff
@@ -27,117 +23,109 @@ const tags = [
     faker.random.word(),
   ]),
 ];
-const generateRule = (ruleName: string, options: InlineRule, groupedBy: string[] = []) => {
-  return {
-    [ruleName]: {
-      rule: {
-        name: ruleName,
-      },
-      options,
-      groupedBy,
-    },
-  };
-};
 
-const getRuleDefinitions = () => {
-  // Rule ID: PAY_HCA_CC
-  // Description: High Cumulative Amount - Inbound - Customer (Credit Card)
-  // Condition: Sum of incoming transactions over a set period of time is greater than a limit of credit card.
+export const getRuleDefinitions = () => {
   const _PAY_HCA_CC = {
-    ruleName: 'PAY_HCA_CC',
+    id: 'PAY_HCA_CC',
+    fnName: 'evaluateTransactionsAgainstDynamicRules',
     groupedBy: ['businessId'],
     options: {
       groupByBusiness: true,
       havingAggregate: AggregateType.SUM,
 
-      direction: 'Inbound',
+      direction: 'inbound',
       excludedCounterpartyIds: ['9999999999999999', '999999******9999'],
 
-      paymentMethods: [PaymentMethod.CreditCard],
+      paymentMethods: [PaymentMethod.credit_card],
       excludePaymentMethods: false,
 
       timeAmount: 7,
       timeUnit: 'days',
 
       amountThreshold: 1000,
     } as TransactionsAgainstDynamicRulesType,
-  };
+  } as InlineRule;
```

Comment:
These uses of `as` make more sense as `satisfies` or the `const something: InlineRule = { ... }` syntax.